src/main/resources/default/templates/biz/jobs/params/codelistentry.html.pasta

```diff
@@ -0,0 +1,14 @@
+<i:arg type="sirius.biz.jobs.params.CodeListEntryParameter" name="param"/>
```

Comment:
I think its better to mention the param name before listing the type as this information is more important to see at a glance.